www/lendingbot.js

```diff
@@ -39,6 +39,41 @@ function updateJson(data) {
 
     updateOutputCurrency(data.outputCurrency);
     updateRawValues(data.raw_data);
+    updateNavbar(data.plugins);
+}
+
+function updateNavbar(plugins) {
+
+    // No plugins enabled. Nothing to do.
+    if (plugins == undefined || plugins.enabled == undefined)
+        return;
+
+    var enabled = plugins.enabled
+    var navbar = $('#navbar-menu')
+    var navbarItems = Array()
+
+    // Build list of navbar items
+    $.each($('#navbar-menu li'), function() {
+        if ($(this).attr('id') != undefined) {
+            navbarItems.push($(this).attr('id').toLowerCase())
+        }
+    });
+
+    // iterate through enabled plugins; look for 'navbar': true
+    $.each(enabled, function(i, val) {
+        var v = val.toLowerCase()
+        if ((v in plugins) && ('navbar' in plugins[v]) && (plugins[v]['navbar'])) {
```

Comment:
am I missing something, in the documentation you mentioned 'webui' here you are looking for 'navbar' ?

Reply:
????? woah. Umm. Lemme investigate. Not sure how that happened. I've got three different git repos for this bot so probably copied wrong.